pkg/util/validation/validate.go

```diff
@@ -100,6 +119,41 @@ func ValidateSample(cfg SampleValidationConfig, userID string, ls []cortexpb.Lab
 	return nil
 }
 
+func ValidateExemplar(userID string, ls []cortexpb.LabelAdapter, e cortexpb.Exemplar) ValidationError {
+	if len(e.Labels) <= 0 {
+		DiscardedExemplars.WithLabelValues(exemplarLabelsMissing, userID).Inc()
+		return newExemplarEmtpyLabelsError(ls, []cortexpb.LabelAdapter{}, e.TimestampMs)
+	}
+
+	if e.TimestampMs == 0 {
+		DiscardedExemplars.WithLabelValues(exemplarTimestampInvalid, userID).Inc()
+		return newExemplarMissingTimestampError(
+			ls,
+			e.Labels,
+			e.TimestampMs,
+		)
+	}
+
+	// Exemplar label length does not include chars involved in
+	// text rendering such as "=,  See spec and const definition.
```

Comment:
[nit]

```suggestion
	// text rendering such as "=". See spec and const definition.
```

Reply:
In this case the comment is trying to show the 3 chars: quote, equals, and comma.  Perhaps something like `such as quotation marks, commas, etc` is clearer.

Reply:
Aha! I didn't get it, sorry. I agree that writing it in words may be clearer 👍